### PR TITLE
Added PersonSchema

### PR DESCRIPTION
**Pull request by katelynsills**
## Description
 Added PersonSchema, changed PrimaryContactUserId to PrimaryContactPersonId

## Test Plan
No tests yet.

### CODE REVIEW

File: server/persons/schema/mongoose-schema.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const PersonSchema = mongoose.Schema({
+	firstName: String,
+	lastName: String,
+	email: String,
+	phone: String,
+})
+
+PersonSchema.methods.getDisplayName = () => {
+	return '${this.firstName} ${this.lastName}'
```
Comment (karenmrose) [inline server/persons/schema/mongoose-schema.js]: This should be `` not '' 😄

Comment (katelynsills) [inline server/persons/schema/mongoose-schema.js]: oh yeah, good catch

File: server/pregnancy-centers/schema/joi-schema.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ const pregnancyCenterSchemaJoi = Joi.object().keys({
 	name: Joi.string(), // change to PRC name
 	notes: Joi.string(),
 	phone: phoneValidator.phone().validate(),
-	primaryContactUserId: Joi.string(),
+	primaryContactPersonId: Joi.string(),
```
Comment (karenmrose) [inline server/pregnancy-centers/schema/joi-schema.js]: This should actually be an objectId. You'll need to import it from mongo, like this:

```
const mongoose = require('mongoose')
const ObjectId = mongoose.Schema.Types.ObjectId
```

Comment (karenmrose) [inline server/pregnancy-centers/schema/joi-schema.js]: NVM this is the joi schema

**Pull request merged**